ci/linux/build-test-x86_64.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# Clone virgil
+cd ..
+git clone https://github.com/titzer/virgil
+cd virgil
+
+# Set up latest version of virgil
+export PATH=$PATH:"$PWD/bin:$PWD/bin/dev:$PWD/test/config"
+./test/configure
+make
+
+# Set up wizard
+cd ../wizard-engine
+make -j
```

Comment:
There is an `x86-64-linux` target in the Makefile that will build only for that platform. It's a small savings.

Reply:
Done